crypto/fipsmodule/FIPS.md

```diff
@@ -6,9 +6,9 @@ A submodule of AWS-LC, referred to here as the “FIPS module”, is periodicall
 
 NIST has awarded the FIPS module of AWS-LC its validation certificate as a Federal Information Processing Standards (FIPS) 140-3, level 1, cryptographic module.
 
-* AWS-LC-FIPS v1.0: certificate [#4631](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4631) - [security policy](./policydocs/140sp4631.pdf)
-* AWS-LC-FIPS v2.0 (dynamic library): certificate [#4759](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4759) - [security policy](./policydocs/140sp4759.pdf)
-* AWS-LC-FIPS v2.0 (static library): certificate [#4816](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4816) - [security policy](./policydocs/140sp4816.pdf)
```

Comment:
should we remove [these](https://github.com/aws/aws-lc/tree/main/crypto/fipsmodule/policydocs) PDF files?